spec/features/step_definitions/rulezilla_dsl_framework_steps.rb

```diff
@@ -73,17 +75,23 @@
 end
 
 step 'all the matching outcomes are :outcomes' do |outcomes|
-  outcomes = outcomes.split(',').map(&:strip).map {|o| o == 'true' ? true : (o == 'false' ? false : o)}
+  outcomes = outcomes.split(',').map(&:strip).map do |o|
+    if o == 'true'
+      true
+    else
+      (o == 'false' ? false : o)
+    end
+  end
   expect(@rule_klass.all(@record)).to match_array outcomes
 end
 
-step ':does_or_does_not raise the exception :exception' do |does_or_does_not, exception|
+step ':does_or_does_not raise the exception :exception' do |does_or_does_not, _exception|
```

Comment:
I'd assume the intention of this test is to check the message against what is passed in during the steps 2 lines below. This looks like it could be a buggy test, revealed thanks to the rubocops.